src/features/usage-consolidation-accordion/usage-consolidation-content-resource/usage-consolidation-content-resource.test.js

```diff
@@ -0,0 +1,116 @@
+import { render } from '@testing-library/react';
+
+import Harness from '../../../../test/jest/helpers/harness';
+
+import UsageConsolidationContentResource from './usage-consolidation-content-resource';
+import { costPerUseTypes } from '../../../constants';
+
+const costPerUseData = {
+  data: {
+    [costPerUseTypes.RESOURCE_COST_PER_USE]:{
+      attributes: {
+        analysis: {
+            cost: 1,
+            costPerUse: 1,
+            usage: 1
+        },
+        parameters: {
+            currency: '',
+            startMonth: '',
+        },
+        usage:{
+            platforms:[
+                {
+                    counts: [3, 6, 16],
+                    isPublisherPlatform: true,
+                    name: "name",
+                    total: 17
+                }
+            ],
+            totals:{}
+          }
+        },
+      },
+  },
+  isFailed: false,
+  isLoaded: true,
+  isLoading: false,
+};
+
+const renderUsageConsolidationContentResource = (props = {}) => render(
+  <Harness>
+    <UsageConsolidationContentResource
+      costPerUseData={costPerUseData}
+      {...props}
+    />
+  </Harness>
+);
+
+describe("Given UsageConsolidationContentResource", () => {
+
+  it('No DOMelement should be rendered when costPerUseData.data.resourceCostPerUse is null', () => {
+    const newCostPerUseData = {
+      ...costPerUseData,
+      data: {
+        ...costPerUseData.data,
+        [costPerUseTypes.RESOURCE_COST_PER_USE]: null
+      }
+    };
+
+    const {container} = renderUsageConsolidationContentResource({
+      costPerUseData: newCostPerUseData
+    });
+
+    expect(container).toBeEmptyDOMElement;
+  });
+
+  it('should render a div with message when isFailed of costPerUseData is true', () => {
+    const newCostPerUseData = {
+      ...costPerUseData,
+      isFailed: true
+    };
+    const { getByText } = renderUsageConsolidationContentResource({
+      costPerUseData: newCostPerUseData
+    });
+
+    expect(getByText('ui-eholdings.usageConsolidation.fullTextRequestUsageTable.noResponse')).toBeDefined();
+  });
+
+  it('should render NoCostPerUseAvailable when noCostPerUseAvailable and isPlatformsDataEmpty are true', () => {
+    const newCostPerUseData = {
+      ...costPerUseData,
+      data:{
+        ...costPerUseData.data,
+        [costPerUseTypes.RESOURCE_COST_PER_USE] : {
```

Comment:
Please don't forget to add a space before opening curly brackets, and on the contrary before the colon there's no need for a space.

```suggestion
      data: {
        ...costPerUseData.data,
        [costPerUseTypes.RESOURCE_COST_PER_USE]: {
```

Reply:
Implemented the changes in latest commit